### PR TITLE
MSTeams module "WebExperiences" - FY2021Q1 refresh

### PR DESCRIPTION
**Pull request by andrewconnell**
- run through all exercise steps in associated MSLearn module to validate good working order
- update code samples to reflect MSLearn HOL exercise refresh & updates
- updated steps & fixed typos where necessary
- updated slides where necessary

### CODE REVIEW

**Comment by islubin** [comment]
Assigning to @lokimeyburg  for review

File: Teams/30 Create Embedded Web Experiences with Tabs for Microsoft Teams/Demos/03-auth-tab/src/app/web/auth-end.html [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
+    crossorigin="anonymous"></script>
   <script src="https://secure.aadcdn.microsoftonline-p.com/lib/1.0.17/js/adal.min.js" crossorigin="anonymous"></script>
 
   <script type="text/javascript">
     microsoftTeams.initialize();
 
     // ADAL.js configuration
     let config = {
-      clientId: "fa792ad0-1f96-4807-b652-929e8efbaf95",
+      clientId: "da0770c2-f800-4ab4-870e-8ffae5d39f6c",
```
Comment (lokimeyburg) [inline Teams/30 Create Embedded Web Experiences with Tabs for Microsoft Teams/Demos/03-auth-tab/src/app/web/auth-end.html:14]: Hmm. Are we sure we want to hardcode this into the repo? Maybe we want to add a placeholder like {----replace with your clientId----}? Unless they are not expected to change it and will use this clientId instead for the purposes of training.

File: Teams/30 Create Embedded Web Experiences with Tabs for Microsoft Teams/Demos/03-auth-tab/src/app/web/auth-start.html [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
+    crossorigin="anonymous"></script>
   <script src="https://secure.aadcdn.microsoftonline-p.com/lib/1.0.17/js/adal.min.js" crossorigin="anonymous"></script>
   <script type="text/javascript">
     microsoftTeams.initialize();
     microsoftTeams.getContext(function (msTeamsContext) {
 
       // ADAL.js configuration
       let config = {
-        clientId: "fa792ad0-1f96-4807-b652-929e8efbaf95",
+        clientId: "da0770c2-f800-4ab4-870e-8ffae5d39f6c",
```
Comment (lokimeyburg) [inline Teams/30 Create Embedded Web Experiences with Tabs for Microsoft Teams/Demos/03-auth-tab/src/app/web/auth-start.html:14]: Ditto for this one. Not sure if this is on purpose or if the developer is expected to replace it with their own clientId.

File: Teams/30 Create Embedded Web Experiences with Tabs for Microsoft Teams/Demos/03-auth-tab/README.md [hunk 3]
```diff
@@ -1,50 +1,118 @@
-# Demo: Add authentication to custom Microsoft Teams channel tabs
+# Learn MSTeams Auth Tabs - Microsoft Teams App
 
-This completed project is the result of the lab exercise **Add authentication to custom Microsoft Teams channel tabs** that is referenced in the [README](./../../README.md) in this repo.
+Generate a Microsoft Teams application.
 
-## Prerequisites
+TODO: Add your documentation here
```
Comment (lokimeyburg) [inline Teams/30 Create Embedded Web Experiences with Tabs for Microsoft Teams/Demos/03-auth-tab/README.md:5]: Another one that I'm not sure what's supposed to go here.

**Review by lokimeyburg — COMMENTED** [review]
* What's the plan for the TODO: Add documentation here sections?
* It's fine for now, but in the future we should recommend the official Teams VSCode Extensions instead of the Yeoman generator 😉
* I wasn't sure about the AAD clientID information. It wasn't clear to me if this was supposed to be replaced by the developer or if this is in fact supposed to be reused. If it's supposed to be replaced, then I suggest we add a placeholder to get them to replace that information
* The manifests look good and up to date 👌

**Pull request merged**